src/v2/view-builder/internals/BaseForm.js

```diff
@@ -38,7 +38,7 @@ export default Form.extend({
     if (input.View) {
       this.add(input.View, {
         options: input.options
-      });
+      }, input.selector);
```

Comment:
looks like you didn't use at all. otherwise could do
```
this.add(input.View, {
  options: input.options,
  selector: input.selector,
});
```

Reply:
was just playing around removed it.